ci-operator/step-registry/telco5g/log-gather/telco5g-log-gather-commands.sh

```
@@ -44,7 +44,16 @@ cp ${SHARED_DIR}/cnf-validations-run.log ${ARTIFACT_DIR}/cnf-validations-run.log
 
 python3 -m venv ${SHARED_DIR}/myenv
 source ${SHARED_DIR}/myenv/bin/activate
-git clone https://github.com/openshift-kni/telco5gci ${SHARED_DIR}/telco5gci
+for attempt in $(seq 1 5); do
+  git clone https://github.com/openshift-kni/telco5gci ${SHARED_DIR}/telco5gci && break
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)